config/crd/bases/cassandra.datastax.com_cassandradatacenters.yaml

```
@@ -348,6 +348,11 @@ spec:
                     - serverSecretName
                     type: object
                 type: object
+              minReadSeconds:
+                description: The number of second the statefulset controller will
+                  wait to see if the pod is in the ready state, 0 is the default value
```

Based on https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#statefulsetspec-v1-apps

Maybe

```suggestion
              minReadSeconds:
                description: Minimum number of seconds for which a newly created pod should be ready without any of its container crashing for it to be considered available. Defaults to 0 (pod will be considered available as soon as it is ready)
```